net/usrsock/usrsock_sockif.c

```diff
@@ -98,43 +98,6 @@ static int usrsock_sockif_setup(FAR struct socket *psock, int protocol)
   int type = psock->s_type;
   int ret;
 
-#ifdef CONFIG_NET_USRSOCK_NO_INET
-  if (domain == PF_INET)
-    {
-      return -ENETDOWN;
-    }
-#endif
-
-#ifdef CONFIG_NET_USRSOCK_NO_INET6
-  if (domain == PF_INET6)
-    {
-      return -ENETDOWN;
-    }
-#endif
```

Comment:
Is this now covered by config options?

Reply:
The option is removed since other side need always check whether the domain is supported.